### PR TITLE
Add broker.subscriber.get_one()

### PR DESCRIPTION
**Pull request by KrySeyt**
# Description

Please include a summary of the change and specify which issue is being addressed. Additionally, provide relevant motivation and context.

Fixes # (issue number)

## Type of change

Please delete options that are not relevant.

- [ ] Documentation (typos, code examples, or any documentation updates)
- [ ] Bug fix (a non-breaking change that resolves an issue)
- [x] New feature (a non-breaking change that adds functionality)
- [ ] Breaking change (a fix or feature that would disrupt existing functionality)
- [ ] This change requires a documentation update

## Checklist

- [x] My code adheres to the style guidelines of this project (`scripts/lint.sh` shows no errors)
- [x] I have conducted a self-review of my own code
- [ ] I have made the necessary changes to the documentation
- [ ] My changes do not generate any new warnings
- [x] I have added tests to validate the effectiveness of my fix or the functionality of my new feature
- [x] Both new and existing unit tests pass successfully on my local environment by running `scripts/test-cov.sh`
- [x] I have ensured that static analysis tests are passing by running `scripts/static-analysis.sh`
- [ ] I have included code examples to illustrate the modifications

### CODE REVIEW

File: faststream/kafka/subscriber/usecase.py [hunk 1]
```diff
@@ -181,16 +184,32 @@ async def close(self) -> None:
...
 
-    async def get_one(self) -> "Optional[KafkaMessage]":
+    async def get_one(self, timeout: float = 5) -> "Optional[KafkaMessage]":
         assert self.consumer, "You should start subscriber at first."
+        assert (  # nosec B101
+            not self.calls
+        ), "You can't use `get_one` method if subscriber has registered handlers."
 
-        assert not self.calls
+        raw_messages = await self.consumer.getmany(timeout_ms=timeout * 1000, max_records=1)
```
Comment (Lancetnik) [inline faststream/kafka/subscriber/usecase.py:193]: Consumer already has getone method, isn't it?

File: faststream/redis/subscriber/usecase.py [hunk 2]
```diff
@@ -450,6 +454,46 @@ def __init__(
...
+            return_msg: Callable[[RedisMessage], Awaitable[RedisMessage]] = (
+                return_input
+            )
+
+            for m in self._broker_middlewares:
+                mid = m(raw_message)
+                await stack.enter_async_context(mid)
+                return_msg = partial(mid.consume_scope, return_msg)
+
+            message = DefaultListMessage(
```
Comment (Lancetnik) [inline faststream/redis/subscriber/usecase.py:489]: Parser? Decoder?

File: faststream/nats/subscriber/usecase.py [hunk 3]
```diff
@@ -418,13 +426,54 @@ def __init__(
...
+            self.subscription = await self._connection.subscribe(
+                subject=self.clear_subject,
+                queue=self.queue,
+                **self.extra_options,
+            )
+
+        raw_message = None
+
+        with anyio.move_on_after(timeout):
+            async for raw_message in self.subscription.messages:
```
Comment (Lancetnik) [inline faststream/nats/subscriber/usecase.py:449]: Why not `self.subscription.get_one`?

Comment (KrySeyt) [reply faststream/nats/subscriber/usecase.py]: Fixed

Comment (Lancetnik) [reply faststream/nats/subscriber/usecase.py]: What's the difference between `get_one` method and `next_msg`? 😃

**Review by Lancetnik — APPROVED** [review]

**Pull request merged**